app/helpers/english.js

```diff
@@ -0,0 +1,4 @@
'insurgent', 'lockup', 'integrate', 'merit', 'intention', 'minnow', 'inventive', 'miser', 'Istanbul', 'Mohawk', 'Jamaica', 'mural', 'Jupiter', 'music', 'leprosy', 'necklace', 'letterhead', 'Neptune', 'liberty', 'newborn', 'maritime', 'nightbird', 'matchmaker', 'Oakland', 'maverick', 'obtuse', 'Medusa', 'offload', 'megaton', 'optic', 'microscope', 'orca', 'microwave', 'payday', 'midsummer', 'peachy', 'millionaire', 'pheasant', 'miracle', 'physique', 'misnomer', 'playhouse', 'molasses', 'Pluto', 'molecule', 'preclude', 'Montana', 'prefer', 'monument', 'preshrunk', 'mosquito', 'printer', 'narrative', 'prowler', 'nebula', 'pupil', 'newsletter', 'puppy', 'Norwegian', 'python', 'October', 'quadrant', 'Ohio', 'quiver', 'onlooker', 'quota', 'opulent', 'ragtime', 'Orlando', 'ratchet', 'outfielder', 'rebirth', 'Pacific', 'reform', 'pandemic', 'regain', 'Pandora', 'reindeer', 'paperweight', 'rematch', 'paragon', 'repay', 'paragraph', 'retouch', 'paramount', 'revenge', 'passenger', 'reward', 'pedigree', 'rhythm', 'Pegasus', 'ribcage', 'penetrate', 'ringbolt', 'perceptive', 'robust', 'performance', 'rocker', 'pharmacy', 'ruffled', 'phonetic', 'sailboat', 'photograph', 'sawdust', 'pioneer', 'scallion', 'pocketful', 'scenic', 'politeness', 'scorecard', 'positive', 'Scotland', 'potato', 'seabird', 'processor', 'select', 'provincial', 'sentence', 'proximate', 'shadow', 'puberty', 'shamrock', 'publisher', 'showgirl', 'pyramid', 'skullcap', 'quantity', 'skydive', 'racketeer', 'slingshot', 'rebellion', 'slowdown', 'recipe', 'snapline', 'recover', 'snapshot', 'repellent', 'snowcap', 'replica', 'snowslide', 'reproduce', 'solo', 'resistor', 'southward', 'responsive', 'soybean', 'retraction', 'spaniel', 'retrieval', 'spearhead', 'retrospect', 'spellbind', 'revenue', 'spheroid', 'revival', 'spigot', 'revolver', 'spindle', 'sandalwood', 'spyglass', 'sardonic', 'stagehand', 'Saturday', 'stagnate', 'savagery', 'stairway', 'scavenger', 'standard', 'sensation', 'stapler', 'sociable', 'steamship', 'souvenir', 'sterling', 'specialist', 'stockman', 'speculate', 'stopwatch', 'stethoscope', 'stormy', 'stupendous', 'sugar', 'supportive', 'surmount', 'surrender', 'suspense', 'suspicious', 'sweatband', 'sympathy', 'swelter', 'tambourine', 'tactics', 'telephone', 'talon', 'therapist', 'tapeworm', 'tobacco', 'tempest', 'tolerance', 'tiger', 'tomorrow', 'tissue', 'torpedo', 'tonic', 'tradition', 'topmost', 'travesty', 'tracker', 'trombonist', 'transit', 'truncated', 'trauma', 'typewriter', 'treadmill', 'ultimate', 'Trojan', 'undaunted', 'trouble', 'underfoot', 'tumor', 'unicorn', 'tunnel', 'unify', 'tycoon', 'universe', 'uncut', 'unravel', 'unearth', 'upcoming', 'unwind', 'vacancy', 'uproot', 'vagabond', 'upset', 'vertigo', 'upshot', 'Virginia', 'vapor', 'visitor', 'village', 'vocalist', 'virus', 'voyager', 'Vulcan', 'warranty', 'waffle', 'Waterloo', 'wallet', 'whimsical', 'watchword', 'Wichita', 'wayside', 'Wilmington', 'willow', 'Wyoming', 'woodlark', 'yesteryear', 'Zulu', 'Yucatan'];
```

Comment:
Is the name `english` (both the variable and the filename) a requirement of the package you are using?  If not it might be better to use something a little more descriptive.  Especially since the seed words would never be translated.

If it is a requirement then that is fine.